### PR TITLE
Epic: Custom Actions phase 2

### PR DESCRIPTION
**Pull request by yuval-hazaz**
Close: #7713 

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-server/src/core/moduleDto/dto/CreateModuleDtoArgs.ts
Comment (overbit) [inline packages/amplication-server/src/core/moduleDto/dto/CreateModuleDtoArgs.ts:1]: file name doesn't match the convention https://www.notion.so/amplication/File-Declaration-Naming-666afe407ff14ed6aa54bcd3dead45a8

File: packages/amplication-server/src/core/moduleDto/dto/DeleteModuleDtoArgs.ts
Comment (overbit) [inline packages/amplication-server/src/core/moduleDto/dto/DeleteModuleDtoArgs.ts:1]: file name doesn't match the convention https://www.notion.so/amplication/File-Declaration-Naming-666afe407ff14ed6aa54bcd3dead45a8

File: packages/amplication-server/src/core/moduleDto/dto/FindManyModuleDtoArgs.ts
Comment (overbit) [inline packages/amplication-server/src/core/moduleDto/dto/FindManyModuleDtoArgs.ts:1]: file name doesn't match the convention https://www.notion.so/amplication/File-Declaration-Naming-666afe407ff14ed6aa54bcd3dead45a8

File: packages/amplication-server/src/core/moduleDto/dto/ModuleDto.ts
Comment (overbit) [inline packages/amplication-server/src/core/moduleDto/dto/ModuleDto.ts:1]: file name doesn't match the convention https://www.notion.so/amplication/File-Declaration-Naming-666afe407ff14ed6aa54bcd3dead45a8

File: packages/amplication-server/src/core/moduleDto/dto/ModuleDtoCreateInput.ts
Comment (overbit) [inline packages/amplication-server/src/core/moduleDto/dto/ModuleDtoCreateInput.ts:1]: file name doesn't match the convention https://www.notion.so/amplication/File-Declaration-Naming-666afe407ff14ed6aa54bcd3dead45a8

**Review by overbit — COMMENTED** [review]
Maybe we should add a linter to ensure conventions are followed

**Review by morhag90 — APPROVED** [review]

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**